apiserver/bundle/bundle_test.go

```diff
@@ -1,26 +1,47 @@
-// Copyright 2015 Canonical Ltd.
+// Copyright 2016 Canonical Ltd.
 // Licensed under the AGPLv3, see LICENCE file for details.
 
-package client_test
```

Comment:
You will need to keep a test in the client to show that client calls still work.